pkg/point/request.go

```diff
@@ -0,0 +1,26 @@
+package point
+
+type CreateRequest struct {
```

Comment:
os outros clients estão assim
```suggestion
type Request struct {
```

Reply:
Ajustado!!